models/afectacion_presupuestal.go

```diff
@@ -13,7 +13,7 @@ import (
 type AfectacionPresupuestal struct {
 	Id                int       `orm:"column(id);pk;auto"`
 	IngresoId         *Ingreso  `orm:"column(ingreso_id);rel(fk)"`
-	RubroId           int       `orm:"column(rubro_id)"`
+	RubroId           string    `orm:"column(rubro_id)"`
```

Comment:
De acuerdo a la L106 de `database/scripts/20220609_141721_inicial_up.sql` faltaría el [`size`](https://beego.vip/docs/mvc/model/models.md#size)

```suggestion
	RubroId           string    `orm:"column(rubro_id);size(38)"`
```